src/components/CalculatePage.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-trailing-spaces */
```

Comment:
- [ ] I can see you have disabled the `eslint` most of the pages. Please try to fix them as they are included just to make your code standard.

Reply:
ok, thanks. I fixed them but for the linebreak, I tried researching but I don't know how to resolve that. Please can you suggest me something?